inc/saitypes.h

```diff
@@ -1128,6 +1128,51 @@ typedef sai_status_t (*sai_bulk_object_remove_fn)(
         _In_ sai_bulk_op_error_mode_t mode,
         _Out_ sai_status_t *object_statuses);
 
+/**
+ * @brief Bulk objects set attributes.
+ *
+ * @param[in] object_count Number of objects to set on attribute
+ * @param[in] object_id List of object ids
+ * @param[in] attr_list List of attributes for every object, one per object.
```

Comment:
Does this allow a different attribute to be set on different objects in the list?

Reply:
yes